LanguageFeatures/Static-access-shorthand/non_ambiguity_A02_t07.dart

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion A postfix expression expression can follow a `?` in a conditional
+/// expression, as in `{e1 ? . id : e2}`. This is not ambiguous with `e1?.id`
+/// since we parse `?.` as a single token, and will keep doing so. It does mean
+/// that `{e1?.id:e2}` and `{e1? .id:e2}` will now both be valid and have
+/// different meanings, where the existing grammar didn’t allow the `?` token to
+/// be followed by `.` anywhere.
+///
+/// @description Checks that spread collections  `....id` is parsed as
+/// `... .id`
+/// @author sgrekhov22@gmail.com
+
+// SharedOptions=--enable-experiment=dot-shorthands
+
+import '../../Utils/expect.dart';
+
+main() {
+  var list = <num>[
+    ....castFrom<num, int>([1, 2, 3])
```

Comment:
Cool! So we can test the actual behavior in a non-error way, too. I hadn't thought of this way to do it.